### ISSUE TITLE
Docker installation can conflict with Fish installation on CentOS7

### ISSUE THREAD

**Issue by leeola** [docker/docker#18138]
## Use the commands below to provide key information from your environment:

docker version: Not installed.
docker info: Not installed.
uname -a: `Linux [removed nodeinfo] 3.14.32-xxxx-grs-ipv6-64 #5 SMP Wed Sep 9 17:24:34 CEST 2015 x86_64 x86_64 x86_64 GNU/Linux`

## Provide additional environment details (AWS, VirtualBox, physical, etc.):

Physical dedicated machine, CentOS 7.1.1503.

## List the steps to reproduce the issue:
1. Install fish shell via the [official centos7 instructions](http://software.opensuse.org/download.html?project=shells%3Afish%3Arelease%3A2&package=fish)
2. Attempt to install Docker via the **"Install without the script"** instructions, [located here](http://docs.docker.com/v1.8/installation/centos/)
3. The step 2 instructions will have you run `yum install docker`, then you'll see the error.

## Describe the results you received:

Yum exited with the following conflict:
```
Transaction check error:
  file /usr/share/fish/completions/docker.fish from install of docker-engine-1.9.1-1.el7.centos.x86_64 conflicts with file from package fish-2.2.0-3.1.x86_64
```

## Describe the results you expected:

No errors from yum.

## Provide additional info you think is important:

You can work around this issue by installing the yum dependencies by hand, and downloading the RPM manually. Once downloaded _(or by using the rpm file url)_, use the `--replacefiles` option on `rpm`, which forces the conflict to be overridden.

**Comment by leeola** [comment]
I have edited my post with the requested info. Again, not a problem with docker, just an annoying build that conflicts with Fish _(or you could blame fish, /shrug)_.

**Comment by thaJeztah** [comment]
Actually, this is already fixed in https://github.com/docker/docker/pull/17144, but not yet used for the RPMs in the 1.9.x release.

Thanks for reporting, @leeola. I'll close this issue as this should be fixed in the next (1.10) release. Apologies for the inconvenience :+1:

**Comment by leeola** [comment]
Appreciate the fix, :+1:
